### ISSUE TITLE
Installation Issue: R package install requires RTools

### ISSUE THREAD

**Issue by yugenechen** [r-windows/docs#10]
I loaded:  Rx64 3.6.3
Upon loading the following packages:  "tis", "mailR", "jsonlite",
all gave a similar warning and error.  Basically, a message states that Rtools is required and is not currently installed.  So, I installed RTools but I still get the same message.  I put in the "PATH="${RTOOLS40_HOME}\\usr\\bin;${PATH}" in a <.environ> file and made sure that the <make()> command works but I still cannot load packages.  Looking at a copy of the response to loading the jsonlite package (as an example) I can see a problem where the system is trying to access <c:/Rtools/mingw_64/bin/gcc> and <c:/Rtools/mingw_64/bin/gcc>; these paths do not exist.  All software is loaded to the "D:" drive and not the "C:" drive.  

1.  Is there a way to fix this by manually assigning the dirve to D: in the paths?
2.  The documentation for R and/or RTools should be updated for this case of installing to an alternate drive (i.e. not the "C:" drive).

### **Commands and response below from RStudio Console:**
> ## Confirm that make.exe is on the PATH
> Sys.which("make")
                                      make 
"d:\\RTools\\rtools40\\usr\\bin\\make.exe" 
> ### "C:\\rtools40\\usr\\bin\\make.exe"
> #
> ## Try to install a package
> install.packages("jsonlite", type = "source")
WARNING: **Rtools is required to build R packages but is not currently installed.** Please download and install the appropriate version of Rtools before proceeding:

https://cran.rstudio.com/bin/windows/Rtools/
trying URL 'https://cran.rstudio.com/src/contrib/jsonlite_1.6.1.tar.gz'
Content type 'application/x-gzip' length 1057910 bytes (1.0 MB)
downloaded 1.0 MB

* installing *source* package 'jsonlite' ...
** package 'jsonlite' successfully unpacked and MD5 sums checked
** using staged installation
** libs
c:/Rtools/mingw_64/bin/gcc  -I"D:/PROGRA~1/R/R-36~1.3/include" -DNDEBUG -Iyajl/api       -D__USE_MINGW_ANSI_STDIO   -O2 -Wall  -std=gnu99 -mtune=core2 -c base64.c -o base64.o
sh: c:/Rtools/mingw_64/bin/gcc: No such file or directory
make: *** [D:/PROGRA~1/R/R-36~1.3/etc/x64/Makeconf:208: base64.o] Error 127
ERROR: compilation failed for package 'jsonlite'
* removing 'D:/Program Files/R/R-3.6.3/library/jsonlite'
Warning in install.packages :
  installation of package ‘jsonlite’ had non-zero exit status

**Comment by jeroen** [comment]
The documentation in this repository is for R 4.0 and rtools40. If you use R 3.6.3 you need to install an older version of rtools: https://cran.r-project.org/bin/windows/Rtools/history.html

In Rtools40 and R 4.0, R will automatically find the compiler if you install it on another drive. But for R 3.x you should install rtools in the default directory.

**Comment by yugenechen** [comment]
Jeroen:

Thank you.  I just came here to update my case.  I clicked on the (older packages) link from the RTools40 page and saw the very advisement you gave - R3.6.3 requires RTools3.5.

https://cran.r-project.org/bin/windows/Rtools/history.html

(Note:  The default page Rtools - CRAN page (cran.r-project.org/bin/windows/Rtools/) brings you to the RTools40 instead of the link above that is more informative.)
